hadoop-ozone/dist/src/main/smoketest/s3/s3_compatbility_check.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# shellcheck disable=SC2086
```

Comment:
Is lack of quotes (flagged by [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)) necessary for the script to work?

Reply:
The situation is the opposite: the quotes are not necessary to make it safe. None of the environment variables are file names and none of them can have spaces. No reason to add quotes.